openfisca_us/data/geography/create_zip_code_dataset.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+
+# Per-ZCTA population dataset
+# ACS 5-year estimates, download URL: https://data.census.gov/cedsci/table?q=DP05%3A%20ACS%20DEMOGRAPHIC%20AND%20HOUSING%20ESTIMATES&g=0100000US%248600000&tid=ACSDP5Y2020.DP05
+
+zcta = pd.read_csv("zcta_2020_population.csv", low_memory=False)
```

Comment:
Can replace lines 6-9, and do similarly for other csvs in this script
```suggestion
zcta = pd.read_csv("zcta_2020_population.csv", low_memory=False, usecols=["DP05_0001E", "NAME"], skiprows=1, header=0, names=["population", "zcta"])
```